michaelplazek/on-track
# UI updates

## Description
- hid the demo pane
- calculate proper size of trainModel window at runtime
- added degrees symbol
- fixed check style errors

## Changes
N/A
## Testing
- Run application. 
- Create and dispatch train from ctc.
- Ensure clock is running.
- View train model UI.

## Additional Information
N/A
